build/pkgs/importlib_metadata/spkg-configure.m4

```diff
@@ -1,3 +1,26 @@
 SAGE_SPKG_CONFIGURE([importlib_metadata], [
   SAGE_PYTHON_PACKAGE_CHECK([importlib_metadata])
+],[
+  # Three of our python packages are backport packages providing
+  # python-3.11 features (see coding_in_python.rst):
+  #
+  #   * importlib_metadata
+  #   * importlib_resources
+  #   * typing_extensions
+  #
+  # These packages are therefore not needed with >=python-3.11. Here
+  # we test for a python minor version component greater than or equal
+  # to 11, and mark this package as "not required" if we succeed.
+  AC_MSG_CHECKING([for >=python-3.11])
```

Comment:
Why does the Python version need a separate check in `spkg-configure.m4`? 
Does the conditionalization in `install-requires.txt` not do the job already? Checking whether `importlib_metadata >=4.13; python_version<'3.11'` is available will be trivially true for Python >= 3.11

Reply:
Oh. I wrote that check before you pointed out that we needed to change install-requires.txt. It's probably redundant now but I have to get flint-3.x packaged before I can run `./configure` now.

Reply:
I had to change the quoting in the macro but it basically works, if you don't care that it says "will use the system foo" rather than "foo is not required" when it skips the package.

Reply:
I think that's fine